### PR TITLE
Add support for RAMB36 cells

### PR DESCRIPTION
**Pull request by mtdudek**
This PR add support for BRAM36 in xc7 fasm.
Also it fixes some bugs found on the way, like incorrect regexes 

Signed-off-by: Maciej Dudek <mdudek@antmicro.com>

### CODE REVIEW

**Comment by mtdudek** [comment]
Added test to fpga-interchange-tests https://github.com/SymbiFlow/fpga-interchange-tests/pull/57

File: fpga_interchange/fasm_generators/xc7.py [hunk 1]
```diff
@@ -377,6 +539,12 @@ def handle_luts(self):
 
             self.luts[key][LutsEnum.from_str(lut_type)] = phys_lut_init
 
+    def handle_lutram(slef):
```
Comment (mkurc-ant) [inline fpga_interchange/fasm_generators/xc7.py:542]: This is not related to RAMB36 so please remove it

**Review by mkurc-ant — CHANGES_REQUESTED** [review]
One minor change requested. Otherwise LGTM

File: fpga_interchange/fasm_generators/xc7.py (hunk 1 above)
Comment (mtdudek) [reply fpga_interchange/fasm_generators/xc7.py]: Removed it form PR
